base/hashing.jl

```diff
@@ -114,3 +114,9 @@ function hash(s::String, h::UInt)
     h += memhash_seed
     ccall(memhash, UInt, (Ptr{UInt8}, Csize_t, UInt32), s, sizeof(s), h % UInt32) + h
 end
+
+function hash(m::Module, h::UInt64)
```

Comment:
Instead of adding this, we should make `objectid` use the module's UUID.

Reply:
I don't think we can do that currently. The UUID is mutable.

Reply:
We will have to solve that --- adding a julia `hash` method doesn't help because the type system and cache won't call it. I suppose this algorithm could be put into `objectid` for now though.

Reply:
Ah, I see you did that already. So this method can just be removed.